WordPress/src/main/res/layout/modal_layout_picker_fragment.xml

```diff
@@ -1,38 +1,104 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:animateLayoutChanges="true"
-    android:orientation="vertical">
+    android:fitsSystemWindows="false"
+    android:focusableInTouchMode="true"
+    app:layout_behavior="@string/bottom_sheet_behavior">
 
-    <include
-        layout="@layout/bottom_sheet_handle_view"
-        android:layout_marginBottom="@dimen/mlp_layouts_handle_margin_bottom"
+    <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/appBarLayout"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content" />
+        android:layout_height="wrap_content"
+        android:background="?attr/categoriesBackground"
+        android:fitsSystemWindows="false">
 
-    <include
-        layout="@layout/modal_layout_picker_header"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content" />
+        <com.google.android.material.appbar.CollapsingToolbarLayout
```

Comment:
Thank you for using `CollapsingToolbarLayout` 💙.

Reply:
Thank you [for suggesting this](https://github.com/wordpress-mobile/WordPress-Android/pull/12482#issuecomment-669743793) :) I wish I had persisted more on using the `CollapsingToolbar` in my initial implementation